hazelcast/src/main/resources/hazelcast-full-example.xml

```diff
@@ -2013,13 +2018,9 @@ https://hazelcast.org/documentation/.
     <quorum name="quorumRuleWithThreeMembers" enabled="true">
         <quorum-size>3</quorum-size>
         <quorum-type>READ_WRITE</quorum-type>
-        <quorum-function-class-name>
-            com.your-package.AbsPresMyQuorum
-        </quorum-function-class-name>
+        <quorum-function-class-name>com.your-package.AbsPresMyQuorum</quorum-function-class-name>
         <quorum-listeners>
-            <quorum-listener>
-                com.your-package.ThreeMemberQuorumListener
-            </quorum-listener>
+            <quorum-listener>com.your-package.ThreeMemberQuorumListener</quorum-listener>
```

Comment:
This string was not allowed to contain whitespaces, so I changed the sample from multi-line to single-line.